makeflow/src/makeflow_catalog_reporter.c

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright (C) 2016- The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file COPYING for details.
+*/
+
+#include "dag.h"
+#include "timestamp.h"
+#include "stringtools.h"
+#include "list.h"
+#include "makeflow_summary.h"
+#include "../../dttools/src/catalog_query.h"
```

Comment:
I believe you can write this simply as `catalog_query.h` and the build system will set the paths correctly.